# This msg file specifies the object 3D estimate including
# (1) object pose (2) the bounding box of the object

pybullet_motoman/Pose3D pose
pybullet_motoman/BoundingBox3D dim
